### PR TITLE
Split code

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js 
please review and help.

Ive tested on a project.

The login-form is not being displayed.

Its not a name conflict.
The file is being loaded as a console.log is being shown in the file but not in the actual directive itself.

### CODE REVIEW

File: src/directives/auth-login-form-directive.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    //                 new appAuthLoginFormFactory(scope, element, attrs);
+    //             }
+    //         });
+    //     }
+    // ]);
+    console.log(module)
+    module.directive('loginForm', [
+        'authLoginFormDirectiveFactory',
+        function (authLoginFormDirectiveFactory) {
+            console.log('s')
```
Comment (wytrych) [inline src/directives/auth-login-form-directive.js]: Could you remove these two `console`s

File: src/constants.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        LOGOUT: '/logout/',
+        LOGIN_REDIRECT_URL: '/',
+        LOGOUT_REDIRECT_URL: '/logout/',
+        AUTH_HEADER_PREFIX: 'Token',
+        ALLOWED_HOSTS: [],
+        COOKIE_PATH: null
+        // Optional settings:
+        // STORAGE_METHOD: 'cookie'
+        // STORAGE_METHOD: 'localStorage'
+        // STORAGE_METHOD: 'noSupport'
```
Comment (KevinEtienne) [inline src/constants.js]: I know this is out-of-scope for this PR but should this and the other comments about implementation should be in the readme instead?

File: src/config.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('angular-token-auth.route-config', [
```
Comment (henrahmagix) [inline src/config.js]: This is named `route-config` but it does more than just the routes, and the filename is `config.js` yet it does routes. Can this be split into `.routes` and `.config` separate file modules?

File: src/constants.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    'use strict';
+
+    var module = angular.module('angular-token-auth.constants', []);
+
+    module.constant('TOKEN_AUTH', {
+        ENDPOINT: '/auth/',
+        LOGIN: '/login/',
+        LOGOUT: '/logout/',
+        LOGIN_REDIRECT_URL: '/',
+        LOGOUT_REDIRECT_URL: '/logout/',
```
Comment (KevinEtienne) [inline src/constants.js]: should this be `/login/` instead?

File: src/config.js (hunk 3 above)
Comment (henrahmagix) [inline src/config.js]: Usually we have a `.routes` file with just the routes in

File: src/directives/auth-login-form-directive.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+(function (angular) {
+    'use strict';
+    var module = angular.module('angular-token-auth.auth-login-form', [
+        'angular-token-auth.auth-login-form-directive-factory',
+    ]);
```
Comment (henrahmagix) [inline src/directives/auth-login-form-directive.js]: I think this file would probably be best combined with something else, in a file that represents the context this is in. I think separating it makes it confusing to understand what it's meant to be doing.

**Comment by henrahmagix** [comment]
I think it'd be best to lessen the impact of these changes by adding a task that combines all the files into the original src file, so projects can still import just that one file and get exactly the same code as before (although perhaps reordered or minified too).

**Comment by Minglee01** [comment]
thats the next step. linting and minifying.

**Comment by Minglee01** [comment]
Closing this as i am going to open a new pr into a feature branch.

**Pull request closed without merging**